AppcuesCapacitor.podspec

```diff
@@ -13,6 +13,6 @@ Pod::Spec.new do |s|
   s.source_files = 'ios/Plugin/**/*.{swift,h,m,c,cc,mm,cpp}'
   s.ios.deployment_target  = '13.0'
   s.dependency 'Capacitor'
-  s.dependency 'Appcues', '~> 2.1'
+  s.dependency 'Appcues', '~> 2.1.1'
```

Comment:
I'm not sure if this is totally necessary for the capacitor plugin, but seems ok and ideal to push to latest with the podspec fix for Expo. fyi @mmaatttt

Reply:
I did it because of your little chat about that on react native side. testing the app it shows the SDK version is correct so i'm good with that.

Reply:
yeah, the reason to consider _not_ doing it, is that `~> 2.1` will work with no changes until if/when we release `3.0` some day, and cocoapods always just gets the latest 2.x.x patch. If we make this change, it will need updated when `2.2` goes out (likely back to `~> 2.2` removing patch)